packages/auth-server/src/index.ts

```diff
@@ -8,10 +8,10 @@ app.use(express.json());
 
 const corsOptions: CorsOptions = {
     origin: [
-        'trezor.io', // production web
-        'sldev.cz', // staging web
-        'localhost', // development web + all desktop
-        'trezoriovpjcahpzkrewelclulmszwbqpzmzgub37gbcjlvluxtruqad.onion', // onion address for production web (Suite does not work here now)
+        'https://suite.trezor.io', // production web
+        /\.sldev\.cz$/, // staging web
+        'http://localhost:8000', // development web
+        'https://trezoriovpjcahpzkrewelclulmszwbqpzmzgub37gbcjlvluxtruqad.onion', // onion address for production web (Suite does not work here now)
```

Comment:
it's not HTTPS over onion
```suggestion
        'http://trezoriovpjcahpzkrewelclulmszwbqpzmzgub37gbcjlvluxtruqad.onion', // onion address for production web (Suite does not work here now)
```

Reply:
Done in 69a2b8584072e289bf020a110e1093185587af24